web-app/js/portal/filter/DateFilter.js

```diff
@@ -123,17 +122,14 @@ Portal.filter.DateFilter = Ext.extend(Portal.filter.Filter, {
     _getDateString: function(newDate) {
 
         if (newDate) {
-            newDate.setHours(this.timezoneCorrection);
```

Comment:
Got the pull request where this was added. Now to analyse it
https://github.com/aodn/aodn-portal/pull/850